website/source/docs/acl/acl-auth-methods.html.md

```diff
@@ -0,0 +1,127 @@
+  privileges assigned to the ACL token that is created. These can be managed
+  with the `consul acl binding-rule` subcommands or the corresponding [API
+  endpoints](/api/acl/binding-rules.html).
+
+## Binding Rules
+
+Binding rules allow an operator to express a systematic way of automatically
+linking [roles](/docs/acl/acl-system.html#acl-roles) and [service
+identities](/docs/acl/acl-system.html#acl-service-identities) to newly created
+tokens without operator intervention.
+
+Successful authentication with an auth method returns a set of trusted
+identity attributes corresponding to the authenticated identity.  Those
+attributes are matched against all configured binding rules for that auth
+method to determine what privileges to grant the the Consul ACL token it will
+ultimately create.
+
+Each binding rule is composed of two portions:
+
+- **Selector** - A logical query that must match the trusted identity
+  attributes for the binding rule to be applicable to a given login attempt.
+  The syntax uses github.com/hashicorp/go-bexpr which is shared with the [API
+  filtering feature](/api/features/filtering.html).  For example:
+  `"serviceaccount.namespace==default and serviceaccount.name!=vault"`
+
+- **Bind Type and Name** - A binding rule can bind a token to a
+  [role](/docs/acl/acl-system.html#acl-roles) or to a [service
+  identity](/docs/acl/acl-system.html#acl-service-identities) by name. The name
+  can be specified with a plain string or the bind name can be lightly
+  templated using [HIL syntax](https://github.com/hashicorp/hil) to interpolate
+  the same values that are usable by the `Selector` syntax. For example:
+  `"dev-${serviceaccount.name}"`
+
+When multiple binding rules match, then all roles and service identities are
+jointly linked to the token created by the login process.
+
+## Overall Login Process
+
+Applications are responsible for exchanging their auth method specific secret
+bearer token for a Consul ACL token by using the login process:
+
+![diagram of auth method login](/assets/images/auth-methods.svg)
+
+1. Applications use the `consul login` subcommand or the [login API
+   endpoint](/api/acl/acl.html#login-to-auth-method) to authenticate to a
+   specific auth method using their local Consul client. Applications provide
+   both the name of the auth method and a secret bearer token during login.
+
+2. The Consul client forwards login requests to the leading Consul server.
+
+3. The Consul leader then uses auth method specific mechanisms to validate the
+   provided bearer token credentials.
+   
+4. Successful validation returns trusted identity attributes to the Consul
+   leader.
+
+5. The Consul leader consults the configured set of binding rules associated
+   with the specified auth method and selects only those that match the trusted
+   identity attributes.
+
+6. Bound roles and service identites are computed. If none are computed the
```

Comment:
kait: Where is 6 and 7 happening? So far steps 1-5 are clear and easy to follow, but I get a little lost here. Does it make sense to have two different lists for this process?